cmake/partition_manager.cmake

```diff
@@ -21,17 +21,15 @@ macro(add_region)
   endif()
 endmacro()
 
-# Get the domain of the current board.
-# In a multi core/SoC build, each core/SoC is a domain as seen from the
-# Partition Manager. Each domain is handled individually by the Partition
-# Manager. Once the Partition Manager has solved all the domains, it creates
-# a global configuration consisting of all domain configurations.
-get_board_without_ns_suffix(${BOARD} domain)
+# The domain of a child image is passed down through the 'DOMAIN' variable.
+if (DOMAIN)
+  set(_DOMAIN _${DOMAIN})
```

Comment:
I would really prefer just to have a single var on this: `DOMAIN`.

Having both `_DOMAIN` and `DOMAIN` doesn't give any advantages, cause you just move the location of the `_`.

like:
```
pm_static_${DOMAIN}.yml
```
vs
```
pm_static${_DOMAIN}.yml
```

unless you expect that `DOMAIN` can be unset.
But then you might get the opposite effect here:
```
${DOMAIN}_PM_HEX_FILE
```
where result would be: `_PM_HEX_FILE`

Reply:
Fixed